tasks.py

```diff
@@ -517,22 +517,27 @@ def pr(c: Context, auto_fix: bool = True, create_pr: bool = True):
 
 
 @task
-def snyk(c: Context):
-    for requirements_file in [
-        "requirements.txt",
-        "dev-requirements.txt",
-        "gpu-requirements.txt",
-        "test-requirements.txt",
-    ]:
-        if Path(requirements_file).exists() is False:
-            raise FileNotFoundError(f"{requirements_file} does not exist")
+def vulnerability_scan(c: Context):
+    requirements_files = Path().parent.rglob("*requirements.txt")
+    top_level_files = [file for file in requirements_files if file.parent == Path()]
+    for requirements_file in top_level_files:
+        if requirements_file.exists() is False:
+            raise FileNotFoundError(f"{requirements_file} does not exist.")
```

Comment:
Is there a need to only get top-level files? And if so, we should just `glob`, not `rglob` 👍 

```python
requirements_files = list(Path(__file__).rglob("*requirements.txt"))
for req in requirements_files:
    ...
```

Reply:
No need to check the file exists anymore either; it won't be found by glob if it doesn't exist.